third_party/web_platform_tests/websockets/binary/001.html

```diff
@@ -2,7 +2,7 @@
 <title>WebSockets: Send/Receive blob, blob size less than network array buffer</title>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
-<script src=../constants.js?pipe=sub></script>
+<script src=../constants.js></script>
```

Comment:
What does having/not having the pipe=sub mean?

Reply:
I'm not so sure, but see John's comments above. I just found it's changed in upstream here: https://chromium-review.googlesource.com/c/chromium/src/+/2848645 and it resolves the timeout issue. I'm still trying to understand the root cause, but we can figure it out later after the merge?